### PR TITLE
Validation: various validation fixes429

### PR DESCRIPTION
**Pull request by Angelines-YG**
fixed duplicate tiles/h1, column-header-missing, alt-text-missing, bookmark-not-found, and indentation.

### CODE REVIEW

File: dynamics-nav-app/change-role.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Change the Role Center
+title: Description on How To Change the Role Center
```
Comment (rflynn27) [inline dynamics-nav-app/change-role.md:2]: On or of?

File: dynamics-nav-app/design-details-item-application.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-    title: Design Details - Item Application 
+    title: Item Application for Design Details
```
Comment (rflynn27) [inline dynamics-nav-app/design-details-item-application.md:2]: I think the way some of these titles are worded can be confusing...I know you're trying to remove duplicate titles, but this reads to me as if Design Details were a software and you talk about the Item Application within. If that is the case, it makes sense! If not, you could consider changing it. Just something to consider!

File: dynamics-nav-app/change-role.md (hunk 1 above)
Comment (Angelines-YG) [reply dynamics-nav-app/change-role.md:2]: I had meant "on" but it sounds better "of". I'll fix it. Thanks!

File: dynamics-nav-app/design-details-item-application.md (hunk 2 above)
Comment (Angelines-YG) [reply dynamics-nav-app/design-details-item-application.md:2]: Gotcha

Comment (edupont04) [reply dynamics-nav-app/design-details-item-application.md:2]: Please be careful here. "Design Details" was conceived as a category of articles many years ago, and the articles' title and H1 start with those two words. So the principle here is "design details about item application", where "item application" is an industry-widely understood term.

Comment (Angelines-YG) [reply dynamics-nav-app/design-details-item-application.md]: @edupont04 Thanks for the clarification!

**Comment by Angelines-YG** [comment]
@edupont04  

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Pull request merged**